ddtrace/llmobs/_integrations/utils.py

```diff
@@ -163,3 +166,41 @@ def get_system_instructions_from_google_model(model_instance):
         elif Part is not None and isinstance(elem, Part):
             system_instructions.append(_get_attr(elem, "text", ""))
     return system_instructions
+
+
+LANGCHAIN_ROLE_MAPPING = {
+    "human": "user",
+    "ai": "assistant",
+    "system": "system",
+}
+
+
+def format_langchain_io(
```

Comment:
These helpers look the same as the ones in `langchain.py` ([ref](https://github.com/DataDog/dd-trace-py/blob/main/ddtrace/llmobs/_integrations/langchain.py#L540-L568)) - can we move those out into the shared _utils.py and reuse them for both integrations?

Reply:
ah yeah i should reuse them for langchain as well, thanks for the callout! i have them currently in the `_integrations/`-level `utils.py` (we have some other google utils there too). just wanna confirm if that's the right spot, or if you're referencing the outer `_utils.py` that has like general SDK utils

Reply:
yup that sounds good to me!